prestoadmin/util/presto_config.py

```diff
@@ -68,21 +68,17 @@ def from_file(config, config_path=None, config_host=None):
     @staticmethod
     def config(host, user):
         config_path = os.path.join(REMOTE_CONF_DIR, CONFIG_PROPERTIES)
-        try:
-            data = StringIO()
-            with settings(host_string='%s@%s' % (user, host)):
-                with hide('stderr', 'stdout'):
-                    temp_dir = run('mktemp -d /tmp/prestoadmin.XXXXXXXXXXXXXX')
-                try:
-                    get(config_path, data, use_sudo=True, temp_dir=temp_dir)
-                finally:
-                    run('rm -r %s' % temp_dir)
-
-            data.seek(0)
-            return PrestoConfig.from_file(data, config_path, host)
-        except Exception as e:
-            _LOGGER.info('Could not find Presto config.')
-            return PrestoConfig(None, config_path, host)
+        data = StringIO()
```

Comment:
As I understand I had two options. So I chose 2. Please see https://github.com/prestodb/presto-admin/pull/309/commits/19dce92cc4d205d61956214e0328af11d8242f2b, there I check if presto is installed before checking its status.